### PR TITLE
Update Contributing Docs - Encourage PRs Even When Stuck

### PR DESCRIPTION
**Pull request by mikaelacaron**
Encourage contributors to submit a PR even if they're stuck on a problem. Discussed with @ahoppen at Swift Community Social!

I wasn't sure exactly what to write or if you want to phrase it differently

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ For more information and instructions, read the GitHub docs on [forking a repo](
 
 Once you've pushed your branch, you should see an option on this repository's page to create a PR from a branch in your fork.
 
+> [!TIP]
+> It's encouraged to submit a PR even if you're stuck on the issue.
```
Comment (ahoppen) [inline CONTRIBUTING.md:83]: What about the following to highlight that we can work together to resolve any issues?

```suggestion
> If you are stuck, it’s encouraged to submit a PR that describes the issue you’re having, e.g. if there are tests that are failing, build failures you can’t resolve, or if you have architectural questions. We’re happy to work with you to resolve those issue.
```

Comment (mikaelacaron) [reply CONTRIBUTING.md:83]: Yes! That sounds good!

**Review by ahoppen — APPROVED** [review]
Thank you, Mikaela! Would you like to also create a similar tip to [sourcekit-lsp](http://github.com/apple/sourcekit-lsp)?

**Comment by ahoppen** [comment]
@swift-ci Please test

**Pull request merged**